spec/index.bs

```diff
@@ -1545,30 +1545,34 @@ interface GPUMapMode {
 
   **Returns:** {{ArrayBuffer}}
 
+  1. If |size| is unspecified and |offset| is less than this.{{[[size]]}}, let |rangeSize| be this.{{[[size]]}} - |offset|. Otherwise, let |rangeSize| be |size|.
```

Comment:
This means that if size is specified, but offset is >= this.[[size]], then we use this.[[size]]. But we only want to use it if size is unspecified. I'll push a commit that does this (along with some indentation fixing to make it work properly)